x-pack/filebeat/module/ibmmq/errorlog/test/AMQERR01.log-expected.json

```diff
@@ -0,0 +1,377 @@
+[
+    {
+        "@timestamp": "2018-10-11T08:46:25.924Z",
+        "ecs.version": "1.0.0",
+        "error.id": "AMQ7125I",
```

Comment:
@webmat This brings up our old discussion around what should go into error and what not? Ingestion error or things from the error log.

Interestingly adding this field broke our tests as we check, that the error object does not exist.

Reply:
Well so far, all concrete uses of `error` have been for pipeline errors.

Accordingly, most error sources have been using `message` for their message. Although looking at all of the different text fields with different shades of error messages (description, explanation, etc) just below here, I'm not sure we can pick one directly for `message`. Is there one of them that's always there?

Anyhow, on to your actual question. In keeping with the actual convention, I'd say we shouldn't use `error.id` here, at least for now. The "userland" or source message going to message, I think the error code should go to `event.code`.

I'm saying this more to follow the current usage convention, more than declaring how it should be used in ECS. I think we can open up that discussion later, and see if we flesh out the usage of `error.*` and put pipeline meta-information such as errors elsewhere in one go, later. One other example of things we could support better is exceptions, stack traces & so on. I think all of that could eventually go in `error.*`, once it's fleshed out.

Reply:
All of them are always in the original message. 
The Description is the most general description of the error. The explanation is more deep. The action describes what a user can do to solve the error.

Reply:
I put it under `ibmmq.errorlog.code` for now.